src/app/catalog/_components/FilterSheet.module.css

```
@@ -100,44 +103,94 @@
+
+/* Same visually-hidden-but-focusable pattern as TriStateCheckbox.module.css's
+   own .input - the native radio stays in the DOM/tab order, the chip itself
+   carries the visible selected/unselected presentation. */
+.chipInput {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  padding: 0;
+  margin: -1px;
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0);
```

Stylelint (17.14.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)